### PR TITLE
CNV-14191: FIPS support RN

### PR DESCRIPTION
**Pull request by sabrinajess**
Re: [CNV-14191](https://issues.redhat.com/browse/CNV-14191)
For 4.9 release notes

Preview: https://deploy-preview-36658--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-9-release-notes.html

### CODE REVIEW

**Review by tiraboschi — APPROVED** [review]

File: virt/virt-4-9-release-notes.adoc [hunk 1]
```diff
@@ -56,6 +56,8 @@ Red Hat is committed to replacing problematic language in our code, documentatio
 
 [id="virt-4-9-installation-new"]
 === Installation
+//CNV-14191 FIPS on CNV
+You can now deploy {VirtProductName} on xref:../installing/installing-fips.adoc#installing-fips[FIPS-enabled clusters].
```
Comment (sjhala-ccs) [inline virt/virt-4-9-release-notes.adoc:60]: Need `*` at the beginning of the sentence to format this as a list item.

**Review by sjhala-ccs — COMMENTED** [review]
One nit, otherwise LGTM

**Pull request merged**